flygym/examples/vision/simple_visual_taxis.py

```diff
@@ -109,6 +110,7 @@ def reset(self, seed=0, **kwargs):
         """See `HybridTurningController.reset`."""
         raw_obs, _ = super().reset(seed=seed)
         self.visual_inputs_hist = []
+        self.arena.reset(self.physics, seed=seed)
```

Comment:
I get an error in the code checker here that `self.arena` is either of type `BaseArena` or `FlatTerrain` but neither of them have a `reset` method.

Is this correct? Or what is actually meant to be happening here?

Reply:
It looks like Alfie added this in 4dd33b19805430fd651ecf97e4b12bf151804c22. Should this be removed @stimpfli?